tests_e2e/test_app.py

```diff
@@ -22,13 +22,9 @@ def driver():
 
 @pytest.fixture(scope="module")
 def test_app():
-    load_dotenv(".env", override=True)
-    trello_board = TrelloBoard(
-        key=os.environ["TRELLO_API_KEY"], token=os.environ["TRELLO_API_TOKEN"]
-    )
-
-    board_id = trello_board.create("test")
-    os.environ["TRELLO_BOARD_ID"] = board_id
+    mongo_test_base_uri = os.environ.get("MONGO_TEST_BASE_URI", default_mongo_uri)
+    mongo_db = "test"
+    os.environ["MONGO_URI"] = f"{mongo_test_base_uri}/{mongo_db}"
```

Comment:
Why do this in two steps with separate MONGO_TEST_BASE_URI and MONGO_URI env vars?